docs/ui/roles.md

```diff
@@ -0,0 +1,138 @@
+| Read-only collaborator | &bull; View flow runs within a workspace. <br> &bull; View deployments within a workspace. <br> &bull; View all Work Queues. <br> &bull; View all Blocks. <br> &bull; View all Notifications. <br> &bull; View workspace settings (handle & description currently). |
+
+## Custom workspace roles
+
+The built-in roles will serve the needs of most users, but your team may need to configure custom roles, giving users access to specific permissions within a workspace.
+
+Custom roles can inherit permissions from a built-in role. This enables tweaks to meet your organization’s needs, while ensuring users can still benefit from Prefect’s default workspace role permission curation as new functionality becomes available.
+
+Custom workspace roles can also be created independent of Prefect’s built-in roles. This option gives workspace admins full control of user access to workspace functionality. However, for non-inherited custom roles, the workspace admin takes on the responsibility for monitoring and setting permisssions for new functionality as it is released.
+
+See [Role permissions](#role-permissions) for details of permissions you may set for custom roles.
+
+After you create a new role, it become available in the organization **Members** page and the **Workspace Sharing** page for you to apply to users.
+
+### Inherited roles
+
+A custom role may be configured as an **Inherited Role**. Using an inherited role allows you to create a custom role using a set of initial permissions associated with a built-in Prefect role. Additional permissions can be added to the custom role. Permissions included in the inherited role cannot be removed.
+
+Custom roles created using an inherited role will follow Prefect's default workspace role permission curation as new functionality becomes available.
+
+To configure an inherited role when configuring a custom role, select the **Inherit permission from a default role** check box, then select the role from which the new role should inherit permissions.
+
+![Creating a custom role for a workspace using inherited permissions in Prefect Cloud](/img/ui/org-inherited-role.png)
+
+## Workspace role permissions
+
+The following permissions are available for custom roles.
+
+### Blocks
+
+| Permission | Description |
+| --- | --- |
+| View blocks | User can see configured blocks within a workspace. |
+| Create, edit, and delete blocks | User can create, edit, and delete blocks within a workspace. Includes permissions of **View blocks**. |
+
+### Concurrency
+
+| Permission | Description |
+| --- | --- |
+| View concurrency limits | User can see configured concurrency limits within a workspace. |		
```

Comment:
To clarify what _type_ of concurrency, maybe something like, "User can see configured task run concurrency limits within a workspace." and, "User can create, edit, and delete task run concurrency limits within a workspace."

Reply:
I have a commit with a fix similar to this that I'll push up after a meeting.